openbas-api/src/main/java/io/openbas/utils/challenge/ChallengeAttemptUtils.java

```diff
@@ -0,0 +1,30 @@
+package io.openbas.utils.challenge;
+
+import io.openbas.database.model.ChallengeAttempt;
+import io.openbas.database.model.ChallengeAttemptId;
+import jakarta.validation.constraints.NotBlank;
+import org.jetbrains.annotations.NotNull;
+
+public class ChallengeAttemptUtils {
+
+  private ChallengeAttemptUtils() {}
+
+  public static ChallengeAttemptId challengeAttemptId(
+      @NotBlank final String challengeId,
+      @NotBlank final String injectStatusId,
+      @NotBlank final String userId) {
+    ChallengeAttemptId challengeAttemptId = new ChallengeAttemptId();
+    challengeAttemptId.setChallengeId(challengeId);
+    challengeAttemptId.setInjectStatusId(injectStatusId);
+    challengeAttemptId.setUserId(userId);
+    return challengeAttemptId;
+  }
+
+  public static ChallengeAttempt challengeAttempt(
```

Comment:
nitpick: rename to buildChallengeAttempt or something like that? Maybe buildDefaultChallengeAttempt even?

Reply:
Fix